packages/effect/src/Stream.ts

```diff
@@ -483,28 +484,62 @@ export const branchAfter: {
 export const broadcast: {
   <N extends number>(
     n: N,
-    maximumLag: number
+    bounded: number | { readonly capacity: number; readonly replay?: number | undefined }
```

Comment:
Why did you rename this?

Reply:
I have addressed additional pubsub options in another PR.

Reply:
Ok, got it, i should focus on shareRefCount in this PR. I don't work today, but will finish when get back to work